app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -44,8 +47,9 @@ class PatientSearchResultItemView(
     context.injector<Injector>().inject(this)
   }
 
-  fun render(model: PatientSearchResultViewModel, currentFacilityId: UUID) {
-    renderPatientNameAgeAndGender(model.fullName, model.gender, DateOfBirth.fromPatientSearchResultViewModel(model, userClock))
+  fun render(model: PatientSearchResultViewModel, currentFacilityId: UUID, searchQuery: String) {
+    val name = highlightPatientName(searchQuery, model, DateOfBirth.fromPatientSearchResultViewModel(model, userClock))
```

Comment:
Since it's just being used by `renderPatientNameAgeAndGender` let's move this into that method.